### PR TITLE
small fixes with missing partials and deprecated method for rails 5

### PR DESCRIPTION
**Pull request by urbanczykd**

### CODE REVIEW

File: app/controllers/kms/public/pages_controller.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-        result   = Kms.template_manager.render(@template.register_id, @externals.
-                                merge(_inner_template: page_result))
+      page_result = Kms.template_manager.render(@page.register_id, @externals)
+      # apply typograph to page content
+      #page_result = EvilFront::Russian.typograph_html(page_result)
+      result   = Kms.template_manager.render(@template.register_id, @externals.
+                              merge(_inner_template: page_result))
 
-      render text: result, layout: false
+      render plain: result, content_type: "text/html"
```
Comment (GarPit) [inline app/controllers/kms/public/pages_controller.rb]: Here should be `render :body` (matches deprecated behavior)

File: app/models/kms/page.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@ def fetch_item!(slug)
     end
 
     def register_id
-      "#{self.class.name.parameterize("_")}_#{id}"
+      (self.class.name << "_" << self.id.to_s).parameterize(separator: "_")
```
Comment (GarPit) [inline app/models/kms/page.rb]: Why need to override this?

File: app/controllers/kms/application_controller.rb [hunk 3]
```diff
@@ -1,11 +1,13 @@
 module Kms
   class ApplicationController < ActionController::Base
+    layout 'kms/devise'
```
Comment (GarPit) [inline app/controllers/kms/application_controller.rb]: It has no problems with layout, do you think this is need refactoring?

https://github.com/webgradus/kms/blob/master/lib/kms/engine.rb#L14

Used this way: https://github.com/plataformatec/devise/wiki/How-To%3A-Create-custom-layouts#application--devise-config

Comment (urbanczykd) [inline app/controllers/kms/application_controller.rb]: tmw I will add some data from rails 5 so we will be able to work it out exactly, and also that will show why I did change it

File: app/controllers/kms/public/pages_controller.rb (hunk 1 above)
Comment (urbanczykd) [inline app/controllers/kms/public/pages_controller.rb]: well any other options ends like that 
screen shot 2017-01-22 at 12 13 22
or like that
screen shot 2017-01-22 at 12 14 53

File: app/models/kms/page.rb (hunk 2 above)
Comment (urbanczykd) [inline app/models/kms/page.rb]: there are actually 2 place where I changed it and reason is this
screen shot 2017-01-22 at 12 21 38

File: app/controllers/kms/application_controller.rb (hunk 3 above)
Comment (urbanczykd) [inline app/controllers/kms/application_controller.rb]: really good idea fixing it works like charm.

Comment (urbanczykd) [inline app/controllers/kms/application_controller.rb]: that was a reason why I add this layout like you point it out to engine configuration  file
screen shot 2017-01-22 at 12 27 15

File: app/models/concerns/kms/compile_templates.rb [hunk 4]
```diff
@@ -9,6 +9,10 @@ module CompileTemplates
       after_commit :compile_templates, on: [:create, :update]
     end
 
+    def register_id
+      (self.class.name << "_" << self.id.to_s).parameterize(separator: "_")
```
Comment (GarPit) [inline app/models/concerns/kms/compile_templates.rb]: Ok for moving it into module, but would like to keep it as it was (only pass named keyword 'separator')

File: app/controllers/kms/public/pages_controller.rb (hunk 1 above)
Comment (GarPit) [inline app/controllers/kms/public/pages_controller.rb]: `html_safe` to the rescue - http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-html

    render html: result.html_safe

**Pull request closed without merging**